cfme/v2v/conversion_hosts.py

```diff
@@ -0,0 +1,236 @@
+    conversion_host_progress = ConversionHostProgress()
+    fill_strategy = WaitFillViewStrategy("15s")
+
+    next_btn = Button("Next")
+    configure_btn = Button("Configure")
+    close_btn = Button("Close")
+
+    @property
+    def is_displayed(self):
+        return self.title.text == "Configure Conversion Host"
+
+    @View.nested
+    class location(View):  # noqa
+        provider_type = BootstrapSelect("providerType")
+        provider_name = BootstrapSelect("provider")
+        cluster = BootstrapSelect("cluster")
+
+        @property
+        def is_displayed(self):
+            return self.provider_type.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class hosts(View):  # noqa
+        hostname = SearchBox(id='host-selection')
+
+        @property
+        def is_displayed(self):
+            return self.hostname.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class authentication(View):  # noqa
+        conv_host_key = HiddenFileInput(locator='.//div[@id="conversionHostSshKey"]/div/input')
+        transformation_method = BootstrapSelect("transformationMethod")
+        vddk_library_path = Input(id='vddk-library-path')
+        vmware_ssh_key = TextInput(locator='.//textarea[@id="vmware-ssh-key-input"]')
+        osp_cert_switch = V2VBootstrapSwitch(label='Verify TLS Certificates for OpenStack')
+        osp_ca_cert = TextInput(locator='.//textarea[@id="openstack-ca-certs-input"]')
+
+        @property
+        def is_displayed(self):
+            return self.conv_host_key.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.configure_btn.click()
+
+    @View.nested
+    class results(View):  # noqa
+        msg = Text('.//h3[contains(@class,"blank-slate-pf-main-action")]')
+
+        @property
+        def is_displayed(self):
+            return self.msg.is_displayed
+
+
+@attr.s
+class ConversionHost(BaseEntity):
+    """Class representing conversion Host
+        Args:
+            target_provider: RHV or OSP
+            provider_name: rhv43 or osp13-ims
+            cluster: Default
+            hostname: host to be configured
+            conv_host_key: /etc/pki/ovirt-engine/keys/engine_id_rsa on RHV-M.
+            transformation_method: vddk or ssh
+            vddk_library_path: url for vddk
+            vmware_ssh_key: /var/lib/vdsm/.ssh/id_rsa on rhv host
+            osp_ca_cert : CA cert
+
+    """
+    target_provider = attr.ib()
+    provider_name = attr.ib()
+    cluster = attr.ib()
+    hostname = attr.ib()
+    conv_host_key = attr.ib()
+    transformation_method = attr.ib()
+    vddk_library_path = attr.ib(default=None)
+    vmware_ssh_key = attr.ib(default=None)
+    osp_cert_switch = attr.ib(default=None)
+    osp_ca_cert = attr.ib(default=None)
+
+    def in_progress(self):
```

Comment:
I am confused by the naming here, if we are waiting for the host to configure shouldn't this be called `wait_for_host_configuration` or something similar? As it stands, I would expect this to return `True` if the host is still undergoing configuration, and `False` otherwise without any `wait_for`. 

Perhaps this can be split into a property, `in_progress` (which by the looks of it is just a call to `view.conversion_host_progress.in_progress(self.hostname)`) and a function `wait_for_host_configuration`

Reply:
Conversely, if you'd like to leave it as is, consider a name change to `in_progress_after_wait`?